src/core/weights.js

```diff
@@ -48,6 +53,42 @@ export function copy(w: Weights): Weights {
   };
 }
 
+/**
+ * Merge multiple Weights together.
+ *
+ * The resultant Weights will have every weight specified by each of the
+ * input weights. If the inputs provide contradictory weights for any node
+ * or edge address, an error will be thrown.
+ */
+export function merge(ws: $ReadOnlyArray<Weights>): Weights {
+  const weights = empty();
+  for (const {nodeWeights, edgeWeights} of ws) {
+    for (const [addr, weight] of nodeWeights.entries()) {
+      const existing = weights.nodeWeights.get(addr);
+      if (existing != null && existing != weight) {
```

Comment:
Why does the `eqeqeq` lint rule allow `existing != weight`?

Reply:
Probably an outdated base branch. Willfix.